Makefile

```diff
@@ -98,7 +98,8 @@ OBJDUMP    := $(MIPS_BINUTILS_PREFIX)objdump
 EMULATOR = mupen64plus
 EMU_FLAGS = --noosd
 
-INC        := -Iinclude -Isrc -Ibuild -I.
+INC        := -Iinclude -Iinclude/libc -Isrc -Ibuild -I.
+build/src/libultra/%.o: INC := -Iinclude/libc -Iinclude/ultra64 -Isrc/libultra
```

Comment:
I think it's just more correct than using `ultra64/...` paths in libultra files, and libultra files ideally shouldn't see any headers from the base include dir (although unfortunately as I mentioned I still couldn't get rid of two headers yet)
It may be better if we moved `src/libultra` to say `lib/libultra` with it's own makefile or something, but that's a bit too far of a departure from what I wanted to achieve for now (but I think doing this later may be good)

Reply:
Yeah I understand the appeal when looking at libultra specifically/separately and I agree libultra should be completely cut off from the rest, my beef is mostly/only that it's confusing when navigating the repo

It would be nice to document this somewhere other than "self-documenting" in the Makefile. I wouldn't be opposed to a `README` file in src/libultra and include/libultra but idk if that would be a popular suggestion

Reply:
Also I'm starting with criticism but I should also congratulate you on a job well done, it's nice to see header cleanup and this looks quite involved <img src="https://cdn.discordapp.com/emojis/695050784655540225.webp?size=128&quality=lossless" height="24" alt="FeelsOKMan">